Code/autopkglib/recipes/__init__.py

```diff
@@ -246,13 +252,36 @@ def to_dict(self, check_only: bool = False) -> dict[str, Any]:
         }
 
     def verify_trust(self) -> bool:
-        """Return True if the recipe trust is correct."""
-        return True
-        # TODO: Implement this
+        """Return True if the recipe trust is verified."""
         # We need to determine if a recipe in the chain is an override and thus contains trust
         # if it contains trust, we then go validate that the trust is correct
         # if there are no overrides, this always returns True (but maybe we print out that we did nothing)
-        # for recipe in self.recipes:
+        for recipe in self.recipes:
+            if recipe.is_override and recipe.trust_info:
+                # We need a way to map the Recipes in our RecipeChain to the parent
+                # recipes in the recipe.trust_info.parent_recipes dictionary.
```

Comment:
The RecipeChain contains a list of identifiers already, and a list of all paths in the chain for all parents.

Reply:
True, but this method allows me to access the `Recipe` objects that are already generated with their `ParentRecipeTrustInfo` objects included in `trust_info`.

Maybe I'm missing something though, happy to take suggestions on a better implementation for this.